cv-basics/convolution.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<ol><li>Please do not mention your names this is a property of a SRA, your names would be there in contributors list </li><li>And use different cells for subtopics, refrain mentioning all the theory in same cell</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@amanchhaparia markdown in one cell won't be an issue.